product_docs/docs/migration_portal/4/03_mp_using_portal/mp_ai_copilot/enable_ai_copilot.mdx

```diff
@@ -7,23 +7,23 @@ To use the AI Copilot, you first must agree to its terms and conditions.
 
 ## Enabling the AI Copilot
 
-After logging in to the [Migration Portal](https://migration.enterprisedb.com/) from the web browser.
-1.  Select **AI Copilot** at the top menu bar. 
-    You can also access the **AI Copilot** from the right-hand pane. This view is only available after selecting a Project and Schema.
+1.  Log in to the [Migration Portal](https://migration.enterprisedb.com/) from the web browser.
+1.  From the top menu bar, select **AI Copilot**. 
+    You can also access the AI Copilot from the right-hand pane. This view is available only after selecting a project and schema.
 1.  Select **Opt in for AI Copilot**.
-    A pop-up window displays the terms and conditions for the usage of the AI Copilot.
-1.  Read the terms and conditions, check the **I have read and agree to the terms and conditions of the agreement**, and close the window.  
+    A pop-up window displays the terms and conditions for the use of the AI Copilot.
+1.  Read the terms and conditions, select **I have read and agree to the terms and conditions of the agreement**, and close the window.  
 
-The AI Copilot is enabled. 
-You can now use the AI Copilot chat to ask questions about the Migration Portal, migration strategies, syntax error resolution, etc. 
-Find usage examples in [How to create good prompts](/migration_portal/4/03_mp_using_portal/mp_ai_copilot/ai_good_prompts/).
+The AI Copilot is enabled.
+You can now use the AI Copilot chat to ask questions about the Migration Portal, migration strategies, syntax error resolution, and so on. 
+For usage examples, see [How to create good prompts](ai_good_prompts/).
 
-## Disabling the AI Copilot 
+## Disabling the AI Copilot
 
 You can disable the AI Copilot at any time. 
 
-After logging in to the [Migration Portal](https://migration.enterprisedb.com/) from the web browser.
-1.  Select your user name at the top right corner of the Migration Portal. 
+1.  Log in to the [Migration Portal](https://migration.enterprisedb.com/) from the web browser.
+1.  At the top-right corner of the Migration Portal, select your user name. 
```

Comment:
@ebgitelman just a bit curious here. 

In other places I have worked, the guideline was to use the action verb first in numbered steps and then refer to the location of the UI element. Reasoning being that action takes precedence over everything else. 

However, I always considered it makes more sense to explain where the UI element is first, so the reader can locate it, and then perform an action with the element. 

I am assuming here at EDB we do the "placing" first, and then the action?